app/styles/pages/admin.scss

```diff
@@ -0,0 +1,3 @@
+.x-scrollable {
```

Comment:
This is not specific to admin. This can be used anywhere. So move to somewhere in partials